.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Contact our Business Team
+    url: https://www.streamx.dev/contact-us.html?utm_source=github&utm_medium=issue&utm_campaign=streamx-examples
+    about: Reach out to our business team for inquiries about StreamX, partnerships, or other opportunities.
+  - name: Request a demo
+    url: https://www.streamx.dev/book-a-demo.html?utm_source=github&utm_medium=issue&utm_campaign=streamx-examples
+    about: Request your demo now to see how StreamX can transform your digital experience!
```

Comment:
The path from StreamX PHP Ingestion Client is quite long to the actual StreamX. I wonder if repo for Ingestion Client is a good place for booking a demo of the whole StreamX solution - but maybe that's on purpose, as a kind of ad :)

Reply:
@stan-r-ds  My idea was to guide particularly interested individuals in the right direction whenever possible. Hence, the two links to contact via streamx.dev. Perhaps it does come across as somewhat promotional. However, if someone has a question but doesn’t know how to ask it, and neither a feature issue nor a bug report feels appropriate, the contact link might just help. And honestly, who would need `streamx-ingestion-php` without StreamX itself? 
Let them take a look at the whole solution.

Reply:
I'm for it, let's broaden ways of getting to know about our fantastic product 🎉